cli/cmd/cluster/apply.go

```diff
@@ -128,22 +128,8 @@ func Apply(contextLogger *log.Entry, options ApplyOptions) error {
 	// Do controlplane upgrades only if cluster already exists and it is not a managed platform.
 	if exists && !c.platform.Meta().Managed {
 		fmt.Printf("\nEnsuring that cluster controlplane is up to date.\n")
-
```

Comment:
This refactor can be committed separately to make the rotation code more highlighted.